### PR TITLE
3rd Party API

### PR DESCRIPTION
**Pull request by CodeBlo**
Added a 3rd party API to count the hits to our API. The hit count can be pulled from the root URL.

### CODE REVIEW

**Review by hamzaakyildiz — APPROVED** [review]
nice api

**Review by asi345 — APPROVED** [review]
Third party api is used without a problem.

**Review by mrabiabrn — APPROVED** [review]

File: practice-app/app.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from flask import Flask, jsonify
+from flask import Flask, jsonify,request
```
Comment (kocasariumut) [inline practice-app/app.py:1]: you could add a space after comma, it could be a better practice

**Review by kocasariumut — APPROVED** [review]
works as intended

**Pull request merged**